src/test/java/org/openmrs/contrib/qaframework/page/CaptureVitalsPage.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.contrib.qaframework.helper.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+/**
+ * Created by tomasz on 25.05.15.
+ */
+public class CaptureVitalsPage extends Page {
+
+	private static final By PATIENT_SEARCH = By.id("patient-search");
+	private static final By FIRST_FOUND_PATIENT = By.cssSelector("i.icon-vitals");
+
+	public CaptureVitalsPage(WebDriver driver) {
+		super(driver);
+	}
+
+	@Override
+	public String getPageUrl() {
+		return "/coreapps/findpatient/findPatient.page?app=referenceapplication.vitals";
+	}
+
+	// public void search(String text) {
+	// WebElement searchField = findElement(PATIENT_SEARCH);
+	// try {
+	// searchField.sendKeys(text);
+	// } catch(Exception e) {
+	// e.printStackTrace();
+	// }
+	// clickOn(FIRST_FOUND_PATIENT);
+	// }
+}
```

Comment:
is CaptureVitalsPage exercised any where?

Reply:
Ofcourse its very excersed i do remember you are the one working on capturevitalsPage :smile:

Reply:
CaptureVitalsPage is not used any where so we need to have it removed among the pages.

Reply:
I do not see that page used any where. Have you had a look at the pages imported within CaptureVitalsTest?

```js
import org.junit.After;
import org.junit.Before;
import org.junit.Test;
import org.junit.experimental.categories.Category;
import org.openmrs.reference.groups.BuildTests;
import org.openmrs.reference.page.ActiveVisitsPage;
import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
import org.openmrs.reference.page.PatientCaptureVitalsPage;
import org.openmrs.uitestframework.test.TestData;
```

Reply:
We would need only pages with tests exercising their functionalities.

Reply:
Resolved thanks